rakefile-dotnet.rb

```diff
@@ -258,6 +268,12 @@
   cmd.parameters "pack #{nuspec} -Version #{version}#{version_suffix} -OutputDirectory #{output}"
 end
 
+desc "create the nuget package with the .NET Standard"
+exec :pack => [:build, output] do |cmd|
+  cmd.command = nuget_command
+  cmd.parameters "pack #{dotnet_nuspec} -Version #{version}-dotnet#{version_suffix} -OutputDirectory #{output}"
+end
+

```

Comment:
I was thinking that we would continue to use the other rakefile to produce the net40-only package, and this rakefile to produce the net40+netstd package.